sdk/appconfiguration/app-configuration/package.json

```diff
@@ -52,14 +59,14 @@
     "prebuild": "npm run clean",
     "pack": "npm pack 2>&1",
     "swagger": "autorest --typescript swagger/swagger.md",
-    "integration-test:browser": "echo skipped",
-    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 180000 --full-trace dist-esm/test/*.spec.js dist-esm/test/**/*.spec.js",
-    "integration-test": "npm run unit-test:node && npm run unit-test:browser",
-    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser",
-    "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "integration-test:browser": "npm run build:test:browser && karma start --single-run",
+    "integration-test:node": "npm run build:test:node && nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 180000 --full-trace dist-esm/test/*.spec.js dist-esm/test/**/*.spec.js",
```

Comment:
Putting `cross-env TEST_MODE=live` would help here while there aren't recordings so that the browser doesn't throw errors when it can't find them (I think this is already done in CI)

Reply:
Sounds like a good idea.

Reply:
Done